charts/argo-server/values.schema.json

```diff
@@ -31,8 +50,15 @@
     "service": {
       "type": "object",
       "properties": {
-        "port": { "type": "integer" }
-      }
+        "port": { "type": "integer" },
+        "ingress": {
+          "oneOf": [
+            { "$ref": "#/definitions/ingressDisabled" },
```

Comment:
whoa this definitions reference thing is cool!

Reply:
JSON schema is so much more useful than Swagger API